### ISSUE TITLE
got some confused result

### ISSUE THREAD

**Issue by shipleyxie** [Tony607/Keras-Trigger-Word#2]
![figure_1](https://user-images.githubusercontent.com/41283865/48264554-d66b0300-e464-11e8-9bc7-601dcfa545d2.png)
i retrain this project  background number is 522(home+factort+street noise), 24 activate audio and 24negatigve audio .
the train log is as foloow:
522/522 [==============================] - 72s 138ms/step - loss: 0.0530 - acc: 0.9998
Epoch 19/20
522/522 [==============================] - 72s 138ms/step - loss: 0.0524 - acc: 0.9998
Epoch 20/20
522/522 [==============================] - 72s 138ms/step - loss: 0.0519 - acc: 0.9998
537/537 [==============================] - 17s 32ms/step
 could pelase give me some guide or suggestion to train the model to a success trigger?
or could you give me some guide on making training data set?
@Tony607

**Comment by Petrimnz** [comment]
You still have this problem ?

**Comment by xavier0102** [comment]
hi, I changed the hot word and I am having this mistake but I do not find why, could you help me? 
![model_trained](https://user-images.githubusercontent.com/51877430/95712879-da61ba00-0c65-11eb-8ff3-253a14f3ab9d.png)
![wrong_result](https://user-images.githubusercontent.com/51877430/95712942-fbc2a600-0c65-11eb-8c7b-959ae48c2b65.png)
and the wrong words go down like hot words.... 
thanks

**Comment by shipleyxie** [comment]
Obviously, your trained model predict wrong result. You need to choose data properly and retrain to get a good model. I suggest you change **epoch** from 40 to 400.
Let me know if this helps.

**Comment by xavier0102** [comment]
hi shipleyxie:
yes, i did it with 400 epoch but do not learn more than 0.85 train and 0.92 test like image, I change the learning rate to 0.005 and the accuracy was better 0.93 in train and 0.92 in test but the result was similar...

**Comment by bluesy7585** [comment]
> hi, I changed the hot word and I am having this mistake but I do not find why, could you help me?
> ![model_trained](https://user-images.githubusercontent.com/51877430/95712879-da61ba00-0c65-11eb-8ff3-253a14f3ab9d.png)
> ![wrong_result](https://user-images.githubusercontent.com/51877430/95712942-fbc2a600-0c65-11eb-8c7b-959ae48c2b65.png)
> and the wrong words go down like hot words....
> thanks

I had similar problem. 
I think its because the learning process is really slow, you going to take many epoch before it gets work.
if you use higher learning rate (e.g 0.0001 > 0.001) and lower drop out rate (e.g 0.8 > 0.5)
just to check if the model can fits the data.
you can get more reasonable result in first few hundreds epoch training.

**Comment by xavier0102** [comment]
Yesss I already resolve that problem, with exactly those tips, I had to wait 3 days training my model, thank you very much.
PS: I had to change the optimizer to RMSprop i think I am going to train with another...
